### PR TITLE
Publish allsections

### PR DESCRIPTION
**Pull request by amjacinto**
Applied certain corrections on the code

### CODE REVIEW

File: app/server/publish.js [hunk 1]
```diff
@@ -19,7 +19,7 @@ Meteor.publish('myClasses', function(flags) {
     // like return Class.find(flags);
 });
 
-Meteor.publish('getViewStates', function(section, flags) {
+Meteor.publish('getViewStates', function(flags) {
     /*
```
Comment (matthewmarcos) [inline app/server/publish.js]: Please remove commented out code

File: app/server/publish.js [hunk 2]
```diff
@@ -19,7 +19,7 @@ Meteor.publish('myClasses', function(flags) {
     // like return Class.find(flags);
 });
 
-Meteor.publish('getViewStates', function(section, flags) {
+Meteor.publish('getViewStates', function(flags) {
```
Comment (matthewmarcos) [inline app/server/publish.js]: Paremove ng flags

File: app/server/publish.js [hunk 3]
```diff
@@ -31,6 +31,19 @@ Meteor.publish('getViewStates', function(section, flags) {
 
         return View.find(flags);
     */
```
Comment (matthewmarcos) [inline app/server/publish.js]: Remove commented-out code

File: app/server/publish.js [hunk 4]
```diff
@@ -31,6 +31,19 @@ Meteor.publish('getViewStates', function(section, flags) {
...
+    var currentUserId = this.userId;
+    var currentUser = Meteor.users.findOne({userId:currentUserId});
```
Comment (matthewmarcos) [inline app/server/publish.js]: Please check [Javascript Conventions](https://github.com/christinejuruena69/128AB5LProject/wiki/Javascript-Conventions)

```
var currentUserId = this.userId,
      currentUser = Meteor.users.findOne({userId:currentUserId});
```

File: app/server/publish.js (hunk 1 above)
Comment (amjacinto) [inline app/server/publish.js]: Should I remove the commented out sample code that you gave?

File: app/server/publish.js [hunk 5]
```diff
@@ -19,18 +19,20 @@ Meteor.publish('myClasses', function(flags) {
...
-
-        check (section, String);
-        check (flags, { something });
-        flags = _.assignInWith(flags, { section });
-
-        return View.find(flags);
-    */
+Meteor.publish('getViewStates', function() {
+    var currentUserId = this.userId;
+    var currentUser = Meteor.users.findOne({userId:currentUserId});
```
Comment (matthewmarcos) [inline app/server/publish.js]: Code convention pa po

File: app/server/publish.js [hunk 6]
```diff
@@ -19,18 +19,19 @@ Meteor.publish('myClasses', function(flags) {
...
-
-        check (section, String);
-        check (flags, { something });
-        flags = _.assignInWith(flags, { section });
-
-        return View.find(flags);
-    */
+Meteor.publish('getViewStates', function () {
+    var currentUserId = this.userId;
+    var currentUser = Meteor.users.findOne({userId : currentUserId});
```
Comment (matthewmarcos) [inline app/server/publish.js:24]: Copy paste the ff to line 23:
Para lang to sa conventions

var currentUserId = this.userId,
    currentUser = Meteor.users.findOne({userId : currentUserId});

**Pull request closed without merging**